docs/content/Popover.md

```diff
@@ -107,25 +111,75 @@ function CaretSelector(props) {
 render(<PopoverDemo />)
 ```
 
-## Component props
+## Props
 
 ### Popover
 
-| Name     | Type              | Default | Description                                                                    |
-| :------- | :---------------- | :-----: | :----------------------------------------------------------------------------- |
-| as       | String            |  'div'  | Sets the HTML tag for the component.                                           |
-| caret    | String            |  'top'  | Controls the position of the caret. See below for the list of caret positions. |
-| open     | Boolean           |  false  | Controls the visibility of the popover.                                        |
-| relative | Boolean           |  false  | Set to true to render the popover using relative positioning.                  |
-| sx       | SystemStyleObject |   {}    | Style to be applied to the component                                           |
-
-#### Caret Positions
-
-The `caret` prop can be one of the following values: `top`, `bottom`, `left`, `right`, `bottom-left`, `bottom-right`, `top-left`, `top-right`, `left-bottom`, `left-top`, `right-bottom`, or `right-top`.
+<PropsTable>
+  <PropsTableRow
+    name="as"
+    defaultValue="div"
+    type="string"
+    description="Sets the underlying HTML tag for the component"
+  />
+  <PropsTableRow
+    name="caret"
+    defaultValue="top"
+    type={
+      <>
+        <InlineCode>top</InlineCode>, <InlineCode>bottom</InlineCode>, <InlineCode>left</InlineCode>, <InlineCode>right</InlineCode>, <InlineCode>bottom-left</InlineCode>, <InlineCode>bottom-right</InlineCode>, <InlineCode>top-left</InlineCode>, <InlineCode>top-right</InlineCode>, <InlineCode>left-bottom</InlineCode>, <InlineCode>left-top</InlineCode>, <InlineCode>right-bottom</InlineCode>, or <InlineCode>right-top</InlineCode>
+      </>
+    }
+    description="Controls the position of the caret"
+  />
+  <PropsTableRow
+    name="open"
+    defaultValue="false"
+    type="boolean"
+    description="Controls the visibility of the popover."
+  />
+  <PropsTableRow
+    name="relative"
+    defaultValue="false"
+    type="boolean"
+    description="Set to true to render the popover using relative positioning. "
+  />
+  <PropsTableSxRow />
+</PropsTable>
 
 ### Popover.Content
 
+<PropsTable>
+  <PropsTableRow
+    name="as"
+    defaultValue="div"
+    type="string"
+    description="Sets the underlying HTML tag for the component"
+  />
```

Comment:
```suggestion
  <PropsTableAsRow
    defaultElementType="div"
  />
```

Reply:
@colebemis This doesn't seem to work with html tags, getting an empty row. Any ideas? ![Screenshot 2022-01-21 at 14 16 36](https://user-images.githubusercontent.com/13340707/150543878-126fe16e-5e78-4bcb-9828-859b939296c2.png)